wlog-zerolog/internal/marshalers/wtracing_spanmodel.go

```diff
@@ -44,6 +44,9 @@ func marshalWTracingSpanModel(evt *zerolog.Event, key string, val interface{}) *
 				encodeSpanModelAnnotations(evt, "cs", "cr", span)
 			}
 		}
+		if tags := span.Tags; tags != nil && len(tags) > 0 {
+			e.Interface(trc1log.SpanTagsKey, tags)
```

Comment:
Can we do something similar to what I recommended on wlog-zap, since I believe `c.Interface` also uses reflection.
There is a `logObjectMarshalerFn` that we define, which we should be able to use for each tag string.